.buildkite/release-pipeline.yaml

```diff
@@ -55,3 +55,18 @@ steps:
           password-env: DOCKERHUB_TOKEN
     env:
       DOCKER_BUILDKIT: "1"
+
+  - label: "Build and publish CPU release image"
+    depends_on: ~
+    if: build.tag != null
+    agents:
+      queue: cpu_queue_postmerge
```

Comment:
@khluu not sure if this `cpu-queue-postmerge` queue is right to build the publish docker image

Reply:
We can use this .. but it should push to an ECR instead of Dockerhub. If the cadence is per release version, we can pull the release image manually from ECR then retag and push to Dockerhub cause it's safer that way (putting Dockerhub token on the CI machine has some risk)

Reply:
@zhouyuan  Can you add this command:
`aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/q9t5s3a7` 
then we can tag it like `public.ecr.aws/q9t5s3a7/vllm-cpu-release-repo:$BUILDKITE_TAG` then push